numba/pythonapi.py

```diff
@@ -143,8 +144,18 @@ def __init__(self, context, builder):
         self.gil_state = Type.int(_helperlib.py_gil_state_size * 8)
         self.py_buffer_t = ir.ArrayType(ir.IntType(8), _helperlib.py_buffer_size)
 
-    def get_env_manager(self, env, env_body):
-        return EnvironmentManager(self, env, env_body)
+    def get_env_manager(self, env, env_body, env_ptr):
+        is_null = cgutils.is_null(self.builder, env_ptr)
```

Comment:
I don't think this method should emit any LLVM code at all. Can't you put the test in one of the callers (if at all necessary)?